src/sidebar/components/HelpPanel.tsx

```diff
@@ -8,9 +8,9 @@ import { VersionData } from '../helpers/version-data';
 import { withServices } from '../service-context';
 import type { SessionService } from '../services/session';
 import { useSidebarStore } from '../store';
-import TabHeader from './ShareDialog/TabHeader';
-import TabPanel from './ShareDialog/TabPanel';
 import SidebarPanel from './SidebarPanel';
+import TabHeader from './Tabs/TabHeader';
```

Comment:
The existing subdirectories in `components` with a capitalized name are names of actual components (eg. `ShareDialog`). There is no component called `Tabs` however. I would either put this component just at the top level of `components`, or name the directory `tabs` to make it clear that it isn't an actual component.